astropy/cosmology/tests/test_cosmology.py

```diff
@@ -1577,6 +1577,21 @@ def test_z_at_value():
     assert allclose(z_at_value(cosmo.distmod, 46 * u.mag),
                     1.9913891680278133, rtol=1e-6)
 
+    # Test array:
+    assert allclose(z_at_value(cosmo.distmod, np.array([46]*4) * u.mag,
+                               zmin=1.5, zmax=2.5, nbins=10000),
+                    1.9913891680278133, rtol=1e-3)
+    assert allclose(z_at_value(cosmo.distmod, np.array([46]*4) * u.mag,
+                               zmin=1.5, zmax=2.5, nbins=10000,
+                               logspace=False, interpolation='linear'),
+                    1.9913891680278133, rtol=1e-3)
+    c = core.WMAP9
```

Comment:
The cosmology package is pretty flat. I'd recommend avoiding calling the internals where possible. See #11345 for how this could error in future (or if that is merged first).

Reply:
It's merged, so this will error.

Reply:
Now ``WMAP9`` is in the ``cosmology.flrw`` submodule but should be imported from just ``cosmology``.